src/components/address.jsx

```diff
@@ -1,8 +1,8 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
 
 class Address extends Component {
```

Comment:
What is the purpose of this component? right now it just looks like it's a heading for the app? If that's the case, we do not need to include that in it's own component.

Reply:
this address.jsx is not in used. deleted